lib/index.js

```diff
@@ -114,27 +115,21 @@ Nanigans.prototype.track = function(track) {
       params.products.value = data.value;
       params.products.sku = data.sku;
       params.products = qs.stringify(params.products);
-      this.load('purchase', params);
+      params.userId ? this.load('purchase', params) : this.load('purchase_no_user_id', params);
       break;
     case 'user':
       switch (event.name) {
-      case 'product':
-        params.sku = data.sku;
-        break;
       case 'add_to_cart':
-        params.products.qty = data.qty;
```

Comment:
@bbeaird not required but if we can still send it we should send it since that's what we are trying to do currently in our integration